From 7cfd1003915aaf8db1109b514b14a1917e35a28a Mon Sep 17 00:00:00 2001
From: Zheng Yejian <zhengyejian1@huawei.com>
Date: Fri, 9 Sep 2022 14:24:25 +0000
Subject: [PATCH 0004/1076] [Huawei] livepatch/arm64: Fix migration thread
 check

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: 187321

--------------------------------

Fix warning introduce the problem.

Fixes: af76fdd54ad2 ("[Huawei] livepatch: Fix warning C_RULE_ID_MAGICNUMBER")
Signed-off-by: Zheng Yejian <zhengyejian1@huawei.com>
---
 arch/arm64/kernel/livepatch.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/kernel/livepatch.c b/arch/arm64/kernel/livepatch.c
index 5ebd4d5279df..6b5bcb491125 100644
--- a/arch/arm64/kernel/livepatch.c
+++ b/arch/arm64/kernel/livepatch.c
@@ -274,7 +274,6 @@ static int do_check_calltrace(struct walk_stackframe_args *args,
 			/* current on this CPU */
 			frame.fp = (unsigned long)__builtin_frame_address(0);
 			frame.pc = (unsigned long)do_check_calltrace;
-		} else if (strncmp(t->comm, "migration/", 10) == 0) {
 		} else if (klp_is_migration_thread(t->comm)) {
 			continue;
 		} else {
-- 
2.17.1

